src/lib/types/domain/song.ts

```diff
@@ -1,14 +1,19 @@
 import type { Timestamp, DocumentReference } from 'firebase/firestore'
 
-export type song = {
+export type Song = {
 	id: string
 	name: string
 	artist: string
-	length: string
 	link: string
+}
+
+export type SongSuggestion = Omit<SuggestedSong, 'id' | 'liked'>
+
+export type SuggestedSong = Song & {
```

Comment:
Wat is het verschil? Is in ieder geval niet duidelijk uit alleen de naam

Reply:
A `SongSuggestion` is all data that is needed by a user to suggest a song. A `SuggestedSong` has been processed and contains all data needed to show it in the suggestions list. Oh auto engels whoops. Ik zal m expliciet maken waar hij gebruikt wordt (1 plek maar)